docs/building_blocks_for_flexible_content.md

```diff
@@ -0,0 +1,571 @@
+      blocks:
+        - type: heading
+          content: Korem ipsum dolor sit
+          path: /landing-page/task
+        - type: statistics
+          title: "Chart to visually represent some data"
+          x_axis_label: "X Axis"
+          y_axis_label: "Y Axis"
+          csv_file: "data_one.csv"
+          data_source_link: https://www.ons.gov.uk/economy/grossdomesticproductgdp/timeseries/ihyq/qna
+          minimal: true
+          minimal_link: /landing-page/task
+  - type: card
+    card_content:
+      blocks:
+        - type: heading
+          content: Korem ipsum dolor sit
+          path: /landing-page/task
+        - type: statistics
+          title: "Chart to visually represent some data"
+          x_axis_label: "X Axis"
+          y_axis_label: "Y Axis"
+          csv_file: "data_one.csv"
+          data_source_link: https://www.ons.gov.uk/economy/grossdomesticproductgdp/timeseries/ihyq/qna
+          minimal: true
+          minimal_link: /landing-page/task
+```
+
+#### Two column layout
+
+A two column layout takes one or two blocks, and depending on the theme (`one_third_two_thirds` or `two_thirds_one_third`), it lays them out horizontally with the first (left) or second (right) block getting twice the width of the other.
+
+##### Example
+
+Theme: one third / two thirds
+
+```yaml
+- type: two_column_layout
+  theme: one_third_two_thirds
+  blocks:
+  - type: side_navigation
+    navigation_group_id: Sidebar
+  - type: blocks_container
+    blocks:
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis
+        molestie, dictum esta, mattis tellus.</p>
+    - type: statistics
+      title: "Chart to visually represent data"
+      x_axis_label: "X Axis"
+      y_axis_label: "Y Axis"
+      csv_file: "data_one.csv"
+      data_source_link: https://www.ons.gov.uk/economy/grossdomesticproductgdp/timeseries/ihyq/qna
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis
+        molestie, dictum esta, mattis tellus.</p>
+```
+
+Theme: two thirds / one third
+
+```yaml
+- type: two_column_layout
+  theme: two_thirds_one_third
+  blocks:
+  - type: blocks_container
+    blocks:
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus.</p>
+    - type: heading
+      content: "Title of content"
+    - type: image
+      theme: full_width
+      src: "landing_page/placeholder/960x640.png"
+      alt: "Placeholder image"
+    - type: govspeak
+      content: |
+        <p>Korem ipsum dolor sit amet, consectetur adipiscing elit. Etiam eu turpis molestie, dictum est a, mattis tellus.</p>
+```
+
+The left column is always populated if only one column is defined.
+A blocks container applied to a column so that everying in the column can be styled relative to one and other.
```

Comment:
I think this sentence is missing some words, I don't quite follow. Also typo in "everying".